src/index.ts

```diff
@@ -12,6 +12,9 @@ export type {Metric, MetricSubscriber} from './visuallyCompleteCalculator';
 
 let calculator: VisuallyCompleteCalculator;
 
+// @ts-ignore
+const startTime = performance?.getEntriesByType?.('navigation')[0]?.activationStart > 0 ? performance?.getEntriesByType?.('navigation')[0]?.activationStart : performance.now()
```

Comment:
Okay, I think I spotted a couple issues here:

1. This line of code is declared at the top level so it will be evaluated immediately when the module is loaded, instead of when `start()` is actually invoked.  That means the value of `performance.now()` is probably not what you expect.

2. You are intercepting the _public_ `start()` method.  In that case, the user is telling us that a navigation is starting right now, and we always will want to override any default start time values.  I would suggest trying to intercept the default value of the `start()` method of the calculator class instead:

https://github.com/dropbox/ttvc/blob/main/src/visuallyCompleteCalculator.ts#L98